### PR TITLE
Explain miserable upper-case-situation

### PR DESCRIPTION
**Pull request by vonloxley**

### CODE REVIEW

**Comment by teverett** [comment]
@KvanTTT ok to merge?

File: tsql/ReadMe.md [hunk 1]
```diff
@@ -4,4 +4,11 @@ This is a community supported grammar file for t-sql, it is cool, yet not very c
...
 but it's not a complete file, so we try hard to stick to the references we could find,
-and bit by bit make it a more complete grammar.
\ No newline at end of file
+and bit by bit make it a more complete grammar.
+
+Use with Antlr 4.7
+------------------
+This grammar uses upper-case key-words only. That means, as sql is case-insensitive, that
+it can’t be used with Antlr 4.7 directly. You need a CharStream that converts to 
+upper-case for the lexer. An implementation for java can be found [on Github](https://github.com/antlr/antlr4/pull/2048/commits/8043e85d178a318172bc94639bd0c131d7e3c60c).
```
Comment (KvanTTT) [inline tsql/ReadMe.md]: I think we should use a link to the file in repository, not to commit. I suggest adding this new stream at least to [antlr4test-maven-plugin](https://github.com/antlr/antlr4test-maven-plugin) repository and use a link from it.

Comment (teverett) [reply tsql/ReadMe.md]: Completely agree.

**Comment by teverett** [comment]
@KvanTTT are we ok to merge this?

**Comment by KvanTTT** [comment]
I don't see any changes. I'm closing it and will wait for ANTLR 4.7.1 with official case insensitive streams support https://github.com/antlr/antlr4/pull/2046

**Pull request closed without merging**